plugins/rbac-backend/src/role-manager/role-manager.test.ts

```diff
@@ -907,7 +916,7 @@ describe('BackstageRoleManager', () => {
       );
       expect(result).toBeFalsy();
       expect(loggerMock.warn).toHaveBeenCalledWith(
-        'Detected cycle dependencies in the Group graph: [["group:default/team-a","group:default/team-c"]]. Admin/(catalog owner) have to fix it to make RBAC permission evaluation correct for group: group:default/team-a',
+        'Detected cycle dependencies in the Group graph: [["group:default/team-a","group:default/team-c"]]. Admin/(catalog owner) have to fix it to make RBAC permission evaluation correct for group: role:default/team-e',
```

Comment:
Looks like 'user:default/mike' doesn't have cycle dependency for role: group:default/team-e.

Reply:
Yes, user:default/mike doesn't have a cycle dependency for role:default/team-e with group:default/team-e. However, there is a cycle dependency in the graph so no matter what role is chosen for mike, there will always be the warning of a cyclic dependency in the graph.